src/main/java/nl/ordina/jobcrawler/model/Skill.java

```diff
@@ -34,7 +37,7 @@
     // Unique set to false for MVP. Set to true in combination with entitymanager if skill column is still needed at some point.
     private String name;
 
-    @ManyToMany(mappedBy = "skills")
+    @ManyToMany(mappedBy = "skills", fetch = FetchType.EAGER)
```

Comment:
Eager fetch type should be avoided as much as possible, is there a reason for this one?